articles/changes.md

```diff
@@ -0,0 +1,245 @@
+
+
+#### Parameters and Dynamic Reconfigure
+
+In ROS 1 global parameters and node-specific dynamic reconfigure parameters are two separate concepts.
+In ROS 2 a unified approach is being used.
+It will be similar to dynamic reconfigure and a node named "global parameter server" will accept requests to set values unconditionally.
+In ROS 1 all this information needs to be polled for changes in ROS 2 changes will be published to notify other entities.
+
+
+#### Threading model
+
+In ROS 1 the developer can only choose between single-threaded execution or multi-threaded execution.
+In ROS 2 more granular execution models are available.
+
+
+#### Graph API
+
+In ROS 1 nodes and topics can be remapped at startup time only.
+In ROS 2 it will be possible to remap those at runtime as well as create aliases.
+
+
+### C and C++
+
+#### Support for real-time
+
+ROS 1 does not support writing real-time code but relies on external frameworks like Orocos.
+In ROS 2 it will be possible to write real-time nodes when using a proper RTOS and with carefully written user code.
+
+
+### C++
+
+#### Node vs. Nodelet
+
+In ROS 1 the API's for nodes and nodelets are different and requires the developer to decide the mapping of nodes to processes at programming time.
+In ROS 2 it will be possible to make this a deploy-time decision.
+It will be recommended to compile each component into a shared library which can then be loaded in a separate process or share the same process with other components (like ROS 1 nodelets).
+
+
+#### Allow multiple nodes per process
+
+In ROS 1 it is not possible to create more than one node in a process.
+This is due to the API itself but also because of internal implementation decisions.
+
+In ROS 2 it will be possible to create multiple nodes in a process.
+
+
+## Tools
+
+### roslaunch
+
+In ROS 1 roslaunch files are defined in xml with very limited capabilities.
+In ROS 2 launch file can be written in Python which enables to use more complex logic like conditionals etc.
+
+
+## Resource lookup
+
+In ROS 1 various resources (packages, messages, plugins, etc.) are looked up by crawling the file system based on the ROS_PACKAGE_PATH.
+This can cause poor performance when the trees in the ROS_PACKAGE_PATH are large, and caching produces inconsistent state.
+
+In ROS 2 resources can be registered at an index at compile time and then be queried efficiently at runtime.
+
+
+## Packaging
+
+### ABI versioning
+
+ROS 1 rebuilds all downstream packages since it assumes ABI incompatibility.
+To avoid this significant overhead a ROS 2 package should be able to declare its ABI to avoid rebuilding downstream packages whenever possible.
+
+
+### Binary packages for Windows
+
+ROS 1 can only be built from source on Windows (which also only works for a few ROS packages and is not supported).
+ROS 2 will provide binary package based on Chocolatey.
```

Comment:
The platforms section only mentions the supported platforms. Here I wanted to emphasize that binary packages will be available. I think this "justifies" a separate subsection.